apps/openchallenges/edam-etl/prepare-python.sh

```diff
@@ -12,4 +12,8 @@ eval "$(pyenv init -)"
 
 pyenv local $PYTHON_VERSION
 poetry env use $PYTHON_VERSION
-poetry install --with prod,dev
\ No newline at end of file
+poetry install --with prod,dev
```

Comment:
The command `nx prepare openchallenges-edam-elt` shows this warning:

```
Group(s) not found: dev (via --with), prod (via --with)
```

Reply:
If this is the command used the `l` and the `t` are in the wrong order. `nx prepare openchallenges-edam-elt`  should be `nx prepare openchallenges-edam-etl` I will run tests post Sage week to confirm. I'm currently trying to finish infrastructure changes for an imaging de-identification challenge that has submission queue opening on Monday.